### ISSUE TITLE
pre-trained model trained on which dataset and saved checkpoint?

### ISSUE THREAD

**Issue by reallynotabot** [keithito/tacotron#125]
Hey, had a question about the [pre-trained model](https://github.com/keithito/tacotron#using-a-pre-trained-model) - which dataset is it trained on, and at what checkpoint/number of steps it was saved?

I trained a model on the LJ Speech dataset till 900k+ steps, following the exact steps and not changing any parameters. The voice doesn't sound as good as the pre-trained model at any checkpoint between 100k-900k.

**Comment by keithito** [comment]
Can you attach some samples? In what way does it not sound as good?

**Comment by reallynotabot** [comment]
@keithito [pre-trained-eval-0 sample](https://soundcloud.com/user-161012414/pretrain-eval-0), and [trained-model-sample](https://soundcloud.com/user-161012414/train-eval-871000-0).

The trained model is at 871k step checkpoint on the LJ Speech dataset, and it sounds more robotic. What checkpoint and dataset is your model at?

**Comment by keithito** [comment]
My checkpoint is at 877k steps on LJ Speech -- but all of the checkpoints after about 500k sounded roughly the same to me.

**Comment by reallynotabot** [comment]
This might be a better example - [pre-trained-model.wav](https://soundcloud.com/user-161012414/pretrained-tacotron-100), [trained-model.wav](https://soundcloud.com/user-161012414/trained-tacotron-100).

The sentence is `Can I have your first and last name?`. 
The trained model messes up the word `have`. Is there some initial random seed in the training that can cause this?
